invokeai/frontend/web/src/features/nodes/store/nodesSlice.ts

```diff
@@ -133,10 +134,6 @@ const nodesSlice = createSlice({
     loadFileEdges: (state, action: PayloadAction<Edge[]>) => {
       state.edges = action.payload;
     },
-    clearNodes: (state) => {
```

Comment:
clearNodes is still being exported. Needs to be removed from there too.

Reply:
yes it is, thanks will fix

Reply:
ok removed it